Packs/ExportIndicators/Integrations/ExportIndicators/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Added support for "mgw", "panosurl" and "bluecoat" output formats.
```

Comment:
You have a typo with mgw, as it should be mwg. Consider using the actual format names such as **McAfee Web Gateway (mwg)**. Please make sure to add this to the readme as well.

Reply:
done